.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,31 +1,11 @@
-<!--- Thank you for opening a pull request! Here are some helpful tips:
-     
-      1. To solicit reviewers: 
-           the @usrse-maintainers are automatically notified when you open this pull request
-           If you need additional reviewers you can:
-               (if you have permission to do so) assign the label "reviewers-needed" 
-               if you are on the usrse slack, post a link to your PR there and ask for reviewers
-
-      2. To get help:
-           you can ask the question directly in this pull request for @usrse-maintainers
-           you can ask a question in the #website channel of usrse.slack.com
-           for important issues, you can @usrse-admin to alert all admins of the repository (use sparingly)
- -->
-
-<!--- Provide a general summary of your changes in the Title above -->
-
 ## Description
-<!--- Describe your changes in detail. -->
+<!--- Describe your changes.  If it fixes an open issue, please link to the issue here. -->
 
-## Motivation and Context
-<!--- Why is this change required? What problem does it solve? -->
-<!--- If it fixes an open issue, please link to the issue here. -->
 
 ## Checklist:
-<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
-<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-- [ ] I have posted the link for the PR in the usrse slack (#website) to ask for reviewers
-- [ ] I have previewed changes locally
-- [ ] I have updated the README.md if necessary
+<!---Check these off after you create the PR --->
+- [ ] I have previewed changes locally or with CircleCI (runs when PR is created)
+- [ ] I have posted the link for the PR in the US-RSE Slack (#website) to ask for reviewers
+
 
-cc @usrse-maintainers
```

Comment:
I don't know that they will be notified, but regardless, that's not quite the right group.  When sorting out the groups, I made usrse-maintainers non-SC members with merge permissions, and then there's a SC group as well.  I think we want people putting the PR on Slack when they're ready for review?  Especially because people need to preview and check tests first (so don't really want to alert folks until it's really ready).  But maybe we do want some @ a group in the template?